doc/nrf/protocols/wifi/wifi_certification.rst

```diff
@@ -11,20 +11,32 @@ The Wi-Fi Alliance® offers a full `certification program <Wi-Fi Certification_>
 The program offers three different paths of certification to Contributor-level `members of the Wi-Fi Alliance <Join Wi-Fi Alliance_>`_.
 Implementer-level members can take advantage of the program by implementing Wi-Fi modules that have been previously certified by a Contributor-level member, for example, one of the `Nordic third-party modules`_.
 
-Qualified solution
+Qualified Solution
 ******************
 
-Nordic Semiconductor supports programs and paths to create qualified solutions that you can use to accelerate the Wi-Fi certification of their end products.
-As a solution provider, it creates qualified solutions and qualified solution variants for its product evaluation platforms based on the nRF70 Series devices and the |NCS|.
+Nordic Semiconductor supports programs and paths to create Qualified Solutions that you can use to accelerate the Wi-Fi certification of their end products.
+As a solution provider, it creates Qualified Solutions and Qualified Solution Variants for its product evaluation platforms based on the nRF70 Series devices and the |NCS|.
 This enables you to use the QuickTrack Wi-Fi certification process.
 
 Wi-Fi certification paths
 *************************
 
 An end product based on an nRF70 Series device can achieve Wi-Fi certification through the QuickTrack or Derivative paths.
-The QuickTrack certification path is applicable for all end products designed from either the Nordic Semiconductor qualified solutions or qualified solution variant evaluation platforms, where limited modifications are allowed.
+The QuickTrack certification path is applicable for all end products designed from either the Nordic Semiconductor Qualified Solutions or Qualified Solution Variant evaluation platforms, where limited modifications are allowed.
 Derivative certification is applicable for end products based on third-party Wi-Fi CERTIFIED™ modules containing an nRF70 Series IC without any modification.
 
 For details about the Wi-Fi certification program and the available paths for the nRF70 Series, read the `Wi-Fi Alliance Certification for nRF70 Series`_ document.
 
 For instructions on how to specifically use the QuickTrack certification path, see the :ref:`wifi_wfa_qt_app_sample` page.
+
+Recertification process after end product software upgrade
+**********************************************************
+
+When a customer’s Wi-Fi CERTIFIED end product requires an update with new software corresponding to a recent |NCS| release, the customer must initiate the Wi-Fi CERTIFIED end product variant process.
```

Comment:
```suggestion
When a customer’s Wi-Fi CERTIFIED end product receives an update with new software corresponding to a recent |NCS| release, the customer must initiate the Wi-Fi CERTIFIED end product variant process.
```

Reply:
I think of "receives" here, to point out better the temporal relation, i.e. first a product receives an update, then a re-certification process must kick off.